sdk/servicebus/azure-messaging-servicebus/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,152 @@
+2. Add [log4j2.xml][log4j2] to your `src/main/resources`.
+
+### Configuring logback
+
+1. Add the dependencies in your pom.xml using ones from the [logging sample pom.xml][LoggingPom] under the "Dependencies required for logback" section.
+2. Add [logback.xml][logback] to your `src/main/resources`.
+
+### Enable AMQP transport logging
+
+If enabling client logging is not enough to diagnose your issues.  You can enable logging to a file in the underlying
+AMQP library, [Qpid Proton-J][qpid_proton_j_apache].  Qpid Proton-J uses `java.util.logging`. You can enable logging by
+creating a configuration file with the contents below.  Or set `proton.trace.level=ALL` and whichever configuration options
+you want for the `java.util.logging.Handler` implementation.  The implementation classes and their options can be found in
+[Java 8 SDK javadoc][java_8_sdk_javadocs].
+
+To trace the AMQP transport frames, set the environment variable: `PN_TRACE_FRM=1`.
+
+#### Sample "logging.properties" file
+
+The configuration file below logs TRACE level output from proton-j to the file "proton-trace.log".
+
+```
+handlers=java.util.logging.FileHandler
+.level=OFF
+proton.trace.level=ALL
+java.util.logging.FileHandler.level=ALL
+java.util.logging.FileHandler.pattern=proton-trace.log
+java.util.logging.FileHandler.formatter=java.util.logging.SimpleFormatter
+java.util.logging.SimpleFormatter.format=[%1$tF %1$tr] %3$s %4$s: %5$s %n
+```
+
+### Reduce logging
+
+One way to decrease logging is to change the verbosity.  Another is to add filters that exclude logs from logger names 
+packages like `com.azure.messaging.servicebus` or `com.azure.core.amqp`.  Examples of this can be found in the XML files 
+in [Configuring Log4J 2](#configuring-log4j-2) and [Configure logback](#configuring-logback).
+
+When submitting a bug, log messages from classes in the following packages are interesting:
+
+* `com.azure.core.amqp.implementation`
+* `com.azure.core.amqp.implementation.handler`
+    * The exception is that the onDelivery message in ReceiveLinkHandler can be ignored.
+* `com.azure.messaging.servicebus.implementation`
+
+
+## Get additional help
+
+Additional information on ways to reach out for support can be found in the [SUPPORT.md][SUPPORT] at the repo's root.
+
+### Filing GitHub issues
+
+When filing GitHub issues, the following details are requested:
+
+* Service Bus configuration / Namespace environment
+    * What tier is the namespace (standard / premium)?
+    * What type of messaging entity is being used (queue/topic)? and it's configuration.
+    * What is the average size of each Message?
+* ServiceBusReceiverClient / ServiceBusProcessorClient environment
+    * What is the machine(s) specs where the client is processing messages?
+    * How many instances are running?
+    * What is the max heap set (i.e., Xmx)?
+    * What is the MaxConcurrentCalls value?
```

Comment:
May need to split this because iirc MaxConcurrentCall does not exist in ServiceBusReceiverClient

Reply:
I restructred it differently. the core idea here is to get the client configuration information from them, so I'll just add some values that we definitely want included if applicable